.travis/.travis_quickstart.sh

```diff
@@ -69,7 +75,7 @@ sleep 30
 for i in $(seq 1 200)
 do
   COUNT_STAR_RES=`curl -X POST --header 'Accept: application/json'  -d '{"sql":"select count(*) from meetupRsvp limit 1","trace":false}' http://localhost:8000/query/sql | jq '.resultTable.rows[0][0]'`
- if [[ "${COUNT_STAR_RES}" =~ ^[0-9]+$ ]]; then
+  if [[ "${COUNT_STAR_RES}" =~ ^[0-9]+$ ]]; then
```

Comment:
this is still not tolerant of failure in curl. The way to do this will be something like
`x=$(curl ...); if [ $? -ne 0 ]; then continue; fi; echo "$x" | jq ...`

Reply:
Added the handling of the error response